Source/SourceKittenFramework/String+SourceKitten.swift

```diff
@@ -54,8 +54,7 @@ extension NSString {
             //
             // A reference to `NSString` is held by every cast `String` along with their views and
             // indices.
-            let cString = string.cStringUsingEncoding(NSUTF8StringEncoding)
-            let string = String(CString: cString, encoding: NSUTF8StringEncoding)!
+            let string = string.mutableCopy() as! String
```

Comment:
@segiddins 
Apple's document said:
> - initWithString:  
> Returns an NSString object initialized by copying the characters from another given string.

But, you can confirm that is incorrect as following:
```swift
import Foundation

let str1 = "Hello, playground" as NSString
let str2 = NSString(string: str1)
let str3 = str1.mutableCopy() as! NSString

let id1 = ObjectIdentifier(str1)
let id2 = ObjectIdentifier(str2)
let id3 = ObjectIdentifier(str3)

id1 == id2 // true
id1 == id3 // false
```